packages/faustwp-core/src/components/WordPressTemplate.tsx

```diff
@@ -103,8 +103,11 @@ export function WordPressTemplate(props: WordPressTemplateProps) {
         '',
       );
 
+      let databaseId = '';
```

Comment:
We're not reassigning this variable. Could we define it instead on line 110 and drop this line?

Reply:
However `queryArgs` is within the control statement in which `databaseId` is being assigned, so defining it there would still allow you to use it within `queryArgs`